__tests__/serverjs/updatecards.test.js

```diff
@@ -2,45 +2,46 @@ const rimraf = require('rimraf');
 const updatecards = require('../../serverjs/updatecards');
 const carddb = require('../../serverjs/cards');
 const examplecards = require('../../fixtures/examplecards');
+const cardutil = require('../../dist/util/Card.js');
```

Comment:
this line is causing tests to fail, is this directory not visible when testing? is it a casing issue?

Reply:
Likely some sort of webpack/rollup issue? That file is created by webpack and/or rollup I believe (during `npm start`). I'm not an expert on any of that or how to get it working in CI.

Reply:
is it just because you are trying to import `../../dist/util/Card.js` instead of `../../dist/util/Card` (no `.js`)?